wayfire.ini

```diff
@@ -151,8 +190,8 @@ idle = swayidle before-sleep swaylock
 
 # Start a terminal
 # https://github.com/alacritty/alacritty
-binding_terminal = <super> KEY_ENTER
-command_terminal = alacritty
+binding_terminal = <super> KEY_ENTER | <ctrl> <alt> KEY_T
+command_terminal = alacritty | gnome-terminal
```

Comment:
I don't think this is a good idea since if the user has both installed, both will be started.